checklist.bs

```diff
@@ -289,37 +317,41 @@ syntax specified in [[DXP#api-action-get-request]].
 The test target host system should respond with a 404 Not Found and a JSON body containing the error
 code `NoSuchFootprint`, as specified in [[DXP#api-error-responses]].
 
-## Test Case 012: Receive Notification of PCF Update ## {#tc012}
+The test target host system must respond with
+- 404 NOT FOUND and a JSON body containing the error code `NoSuchFootprint`,
+    as specified in [[DXP#api-error-responses]], or
+
+- 400 BAD REQUEST and a JSON body containing an error, as specified in
```

Comment:
Likewise, always returning a 400 would be conformant behavior, effectively turning the behavior and this endpoint in general into an "OPTIONAL" one.

Reply:
If the endpoint would be "Optional" a solution would not have to implement it. This would break existing clients. In the spec changes we introduces the "Http-400 => go to async" approach to explicitly avoid this situation. Endpoint needs to be there and to return valid answers (in respect to the specification).

Reply:
In the absence of 2xx codes for signalling async processing, using 400 for this one is in line with list footprints. 

Also for this one, a 202 (Accepted) would be a good solution, I think.